src/homepageExperience/containers/HomepageContainer.tsx

```diff
@@ -156,18 +156,20 @@ export const HomepageContainer: FC = () => {
                         </div>
                       </Link>
                     </ResourceCard>
-                    <ResourceCard style={cardStyle}>
-                      <Link
-                        to={javaScriptNodeLink}
-                        style={linkStyle}
-                        onClick={logNodeJSWizardClick}
-                      >
-                        <div className="homepage-wizard-language-tile">
-                          <h5>Node.js</h5>
-                          {NodejsIcon}
-                        </div>
-                      </Link>
-                    </ResourceCard>
+                    {!isFlagEnabled('ioxOnboarding') && (
+                      <ResourceCard style={cardStyle}>
```

Comment:
Not following this either - we're not showing the JS/Node onboarding in IOx?

Reply:
Correct, this won't be shown for IOx